src/Mutation/PostObjectUpdate.php

```diff
@@ -160,7 +160,7 @@ public static function mutate_and_get_payload( $post_type_object, $mutation_name
 			/**
 			 * Throw an exception if the post failed to update
 			 */
-			if ( 0 === $post_id || is_wp_error( $updated_post_id ) ) {
+			if ( ! $post_id || is_wp_error( $updated_post_id ) ) {
```

Comment:
it should be `0 === $updated_post_id`. https://developer.wordpress.org/reference/functions/wp_update_post/